pkg/etcdutil/etcdutil.go

```diff
@@ -123,7 +125,17 @@ func DoOpsInOneCmpsTxnWithRetry(cli *clientv3.Client, cmps []clientv3.Cmp, opsTh
 	ctx, cancel := context.WithTimeout(cli.Ctx(), DefaultRequestTimeout)
 	defer cancel()
 	tctx := tcontext.NewContext(ctx, log.L())
+
+	failpointCount := 0
 	ret, _, err := etcdDefaultTxnStrategy.Apply(tctx, etcdDefaultTxnRetryParam, func(t *tcontext.Context) (ret interface{}, err error) {
+		failpoint.Inject("ErrNoSpace", func(val failpoint.Value) {
+			maxCount := val.(int)
+			if failpointCount < maxCount {
```

Comment:
some days ago I've learned we could use `3*return()` instead of using extra variable

https://github.com/pingcap/failpoint#key-concepts `[<percent>%][<count>*]<type>[(args...)][-><more terms>]
`

Reply:
https://github.com/pingcap/dm/pull/1033/commits/61729fcb6ec26d71babd5f88df343b9760bf5266